tcms/testruns/static/testruns/js/get.js

```diff
@@ -19,23 +25,30 @@ $(document).ready(() => {
 
     const testRunId = $('#test_run_pk').data('pk')
 
-    $('#status_button').on('switchChange.bootstrapSwitch', (_event, state) => {
-        if (state) {
-            jsonRPC('TestRun.update', [testRunId, { 'stop_date': null }], () => {
-                $('.stop-date').html('-')
-                $('#test_run_pk').parent('h1').css({ 'text-decoration': 'none' })
-            })
-        } else {
-            const timeZone = $('#clock').data('time-zone')
-            const now = currentTimeWithTimezone(timeZone)
-
-            jsonRPC('TestRun.update', [testRunId, { 'stop_date': now }], testRun => {
-                const stopDate = moment(testRun.stop_date).format("DD MMM YYYY, HH:mm a")
-                $('.stop-date').html(stopDate)
-                $('#test_run_pk').parent('h1').css({ 'text-decoration': 'line-through' })
-            })
-        }
-    })
+    $('#start-button').on('click', function () {
+        const timeZone = $('#clock').data('time-zone')
+        const now = currentTimeWithTimezone(timeZone)
+
+        jsonRPC('TestRun.update', [testRunId, { 'start_date': now }], testRun => {
+            const startDate = moment(testRun.start_date).format("DD MMM YYYY, HH:mm a")
+            $('.start-date').html(startDate);
+            $(this).hide();
+        })
+
+    });
+
+    $('#stop-button').on('click', function () {
+        const timeZone = $('#clock').data('time-zone')
+        const now = currentTimeWithTimezone(timeZone)
+
+        jsonRPC('TestRun.update', [testRunId, { 'stop_date': now }], testRun => {
+            const stopDate = moment(testRun.stop_date).format("DD MMM YYYY, HH:mm a")
+            $('.stop-date').html(stopDate);
+            $(this).hide();
```

Comment:
Still not updating title like it used to be. See https://github.com/kiwitcms/Kiwi/pull/2328#discussion_r607568638